markdown/tables/olympian-sagittal-extension-extreme-precision-accidental-diacritics.md

```diff
@@ -1,9 +1,12 @@
 Olympian Sagittal extension (extreme precision) accidental diacritics (U+E3F4–U+E3F7)
 =====================================================================================
 
-*Reserved for future use.*
+| **Glyph** | **Description** | **Glyph** | **Description**
+| :-------: | --------------- | :-------: | ---------------
+|<span class="bravura_large">&#xe3f4;</span> | **U+E3F4**<br/>*accSagittal1MinaUp*<br/>1 mina up, 1/(5⋅7⋅13)-schismina up, 0.42 cents up | <span class="bravura_large">&#xe3f5;</span> | **U+E3F5**<br/>*accSagittal1MinaDown*<br/>1 mina down, 1/(5⋅7⋅13)-schismina down, 0.42 cents down
+|<span class="bravura_large">&#xe3f6;</span> | **U+E3F6**<br/>*accSagittal2MinasUp*<br/>2 minas up, 65/77-schismina up, 0.83 cents up | <span class="bravura_large">&#xe3f7;</span> | **U+E3F7**<br/>*accSagittal2MinasDown*<br/>2 minas down, 65/77-schismina down, 0.83 cents down
 
 Implementation notes
 ---------------------
 
-This range is reserved for the future definition of four glyphs, representing alterations of one and two 455 or 65:77 schisminas. These schisminas are approximately 0.4 cents.
\ No newline at end of file
+These four glyphs represent alterations of one and two 455 or 65:77 schisminas. These schisminas are approximately 0.4 cents.
```

Comment:
@dspreadbury This implementation note should be deleted as well. Thank you again.